### ISSUE TITLE
Some codelens don't work in WSL2

### ISSUE THREAD

**Issue by fbricon** [redhat-developer/intellij-quarkus#856]
The `Open template` code lens from a Java file, or the `Open type` header code lens in a qute template don't work, when the project is opened from a WSL2 environment. No exceptions. Just don't work.

The REST endpoint codelens work though.

Might be something related to #735, although sneakier.

**Comment by angelozerr** [comment]
@fbricon with your last fix, have you again this problem?

**Comment by fbricon** [comment]
Yes it still exists. It looks like the problem might be on the LS side, as intellij-quarkus sends the proper URL. But I need to figure out how to debug the LS
